sdk/monitor/azure-monitor-ingestion/README.md

```diff
@@ -0,0 +1,179 @@
+
+```java readme-sample-createLogsIngestionAsyncClient
+DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+LogsIngestionAsyncClient asyncClient = new LogsIngestionClientBuilder()
+        .endpoint("<data-collection-endpoint>")
+        .credential(tokenCredential)
+        .buildAsyncClient();
+```
+### Uploading logs
+
+For examples on how to upload logs, see the [Examples](#examples) section.
+
+## Key concepts
+
+### Data Collection Endpoint
+
+Data Collection Endpoints (DCEs) allow you to uniquely configure ingestion settings for Azure Monitor. [This 
+article][data_collection_endpoint] provides an overview of data collection endpoints including their contents and 
+structure and how you can create and work with them.
+
+### Data Collection Rule
+
+Data collection rules (DCR) define data collected by Azure Monitor and specify how and where that data should be sent or
+stored. The REST API call must specify a DCR to use. A single DCE can support multiple DCRs, so you can specify a
+different DCR for different sources and target tables.
+
+The DCR must understand the structure of the input data and the structure of the target table. If the two don't match,
+it can use a transformation to convert the source data to match the target table. You may also use the transform to
+filter source data and perform any other calculations or conversions.
+
+For more details, refer to [Data collection rules in Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/essentials/data-collection-rule-overview).
+
+### Log Analytics Workspace Tables
+
+Custom logs can send data to any custom table that you create and to certain built-in tables in your Log Analytics 
+workspace. The target table must exist before you can send data to it. The following built-in tables are currently supported:
+
+- [CommonSecurityLog](https://docs.microsoft.com/azure/azure-monitor/reference/tables/commonsecuritylog)
+- [SecurityEvents](https://docs.microsoft.com/azure/azure-monitor/reference/tables/securityevent)
+- [Syslog](https://docs.microsoft.com/azure/azure-monitor/reference/tables/syslog)
+- [WindowsEvents](https://docs.microsoft.com/azure/azure-monitor/reference/tables/windowsevent)
+
+## Examples
+
+- [Upload custom logs](#upload-custom-logs)
+- [Upload custom logs with max concurrency](#upload-custom-logs-with-max-concurrency)
+
+### Upload custom logs
+
+```java readme-sample-uploadLogs
+DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+LogsIngestionClient client = new LogsIngestionClientBuilder()
+        .endpoint("<data-collection-endpoint")
+        .credential(tokenCredential)
+        .buildClient();
+
+List<Object> logs = getLogs();
+UploadLogsResult result = client.upload("<data-collection-rule-id>", "<stream-name>", logs);
+System.out.println("Logs upload result status " + result.getStatus());
+```
+
+### Upload custom logs with max concurrency
```

Comment:
Should this example add explanation docs on what max concurrency does? My initial guess is parallel uploading but this could also be a service feature